workflow-service/src/main/java/com/redhat/parodos/workflow/execution/dto/WorkStatusResponseDTO.java

```diff
@@ -47,6 +47,8 @@ public class WorkStatusResponseDTO {
 
 	private WorkStatus status;
 
+	private String message;
```

Comment:
`@JsonInclude(JsonInclude.Include.NON_EMPTY)`  ?

Reply:
The API contract/spec defines that the response contains a `message` field, this field shall be present for parodos to fulfil the contract.
This way, the API consumer will not have to worry about non-existing key when processing the response

Reply:
I think it can be optional. we can address it later